modules/git/repo_license_gogit.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
```

Comment:
I don't think it's suitable to put the feature in this sub package.

Reply:
Is `HookPostReceive` the right place?🤔

Reply:
It could be a standalone package under modules or services